.travis.yml

```diff
@@ -5,14 +5,14 @@ matrix:
     include:
         - php: 5.3
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
-        - php: 7.0
-          env: LINT_TEST_CASES=1
```

Comment:
Because I dumb copy pasted the 1.x version plus fix into the master version of the file so it gets tested, after that things will put Back in order